web/profiles/custom/nysenate/config/local/reroute_email.settings.yml

```diff
@@ -1,7 +1,7 @@
 _core:
   default_config_hash: gbxsfZiC2MMjzyiqbCPrKXXxlhQDa0pMNs0FSt_33ds
 enable: true
-address: nys.senate.dev@gmail.com
+address: ''
```

Comment:
I'm happy to revert this, but I wanted to note that I made this change just for the local config split only, not multidev or any other env. Since I don't get emails to nys.senate.dev@gmail.com, this default would be helpful for me when working locally.